linux_os/guide/system/software/integrity/software-integrity/aide/aide_disable_silentreports/oval/ubuntu2004.xml

```diff
@@ -0,0 +1,22 @@
+<def-group>
```

Comment:
Please, rename the file to `shared.xml`. There are already product restrictions defined in `rule.yml`.